src/datasources/db/entities/row.entity.ts

```diff
@@ -9,6 +9,6 @@ export type Row = z.infer<typeof RowSchema>;
  */
 export const RowSchema = z.object({
   id: z.number().int(),
-  created_at: z.date(),
-  updated_at: z.date(),
+  created_at: z.coerce.date(),
+  updated_at: z.coerce.date(),
```

Comment:
This was changed because we store plain JSON in the cache, so the dates are stored as date-formatted strings, which need to be coerced.